page-policy.php

```diff
@@ -25,10 +25,8 @@
 </ul>
 ownCloud Desktop Client:
 <ul>
-	<li>1.6.x</li>
-	<li>1.7.x</li>
-  <li>1.8.x</li>
-	<li>2.0.x</li>
+	<li>2.2.4</li>
```

Comment:
@guruz According to @michaelstingl in https://github.com/owncloud/documentation/issues/2961only the latest stable (2.3.x) is supported. Not sure if this also relevant for this security policy.